packages/mobile/src/index.tsx

```diff
@@ -6,6 +6,7 @@ import { Navigation } from '~/navigation/navigation';
 import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
 import { useColorScheme } from '~/hooks/hooks';
+import './localization/i18n';
```

Comment:
Given that we don't use default exports I assume it should be something like this?

```suggestion
import { i18n } './localization/localization';
```

Reply:
This example was in documentation. Other way it doesn't work .